package.json

```diff
@@ -100,7 +100,8 @@
     }
   },
   "dependencies": {
-    "dotenv": "8.2.0"
+    "dotenv": "8.2.0",
+    "gatsby-transformer-json": "^3.3.0"
```

Comment:
We have a, somewhat complicated, setup in which our Docker images are created: https://github.com/freeCodeCamp/freeCodeCamp/blob/main/server.Dockerfile#L16 so could you move this to the client dir?  Otherwise it will become part of the api image.